src/spec2nexus/writer.py

```diff
@@ -18,11 +18,11 @@
 import h5py
 import numpy as np
 
-import eznx
+import spec2nexus.eznx as eznx
 import spec2nexus
-import spec
-import utils
-
+import spec2nexus.spec as spec
+import spec2nexus.utils as spec
```

Comment:
Sure about this?  Why not instead:
```
import spec2nexus.utils as utils
```

might by why code coverage dropped by 5%